### PR TITLE
#2401 update some rpe emails

### PR DESCRIPTION
**Pull request by stenington**
Update some RPE email titles and add the official/unofficial disclaimer to RPE invitation emails.

### CODE REVIEW

File: app/views/event_mailer/invite.en.html.erb [hunk 1]
```diff
@@ -25,6 +25,17 @@
...
         </td>
       </tr>
+      <tr>
+        <td class="content-block">
+          Your event will be confirmed as either a Regional Pitch Event or Celebration Event
+          after <%= ImportantDates.rpe_officiality_finalized.strftime("%B %-d") %>. Please sign
+          into your account to verify the status of your event after that date. For more
+          information about pitch events, please check out the
+          <%= link_to "Technovation FAQ",
+            "https://iridescentsupport.zendesk.com/hc/en-us/articles/115002731414-All-about-Regional-Pitch-Events"
```
Comment (shaunxp20) [inline app/views/event_mailer/invite.en.html.erb]: Hmm, I think this link is broken? 🤔 🔗 🙅‍♂ 😄

Comment (stenington) [reply app/views/event_mailer/invite.en.html.erb]: It sure is! It's the link from the doc, but not working. @rsgonzal is this a page that is going to eventually exist, or do we need a new url?

Comment (rsgonzal) [reply app/views/event_mailer/invite.en.html.erb]: @stenington team accidentally switched the link. Here is the new one: https://iridescentsupport.zendesk.com/hc/en-us/sections/360007469154-Regional-Pitch-Events

**Comment by stenington** [comment]
@shaunxp20 How do you feel about this one with a corrected link? Thanks for catching that!

**Pull request merged**